mysql-test/suite/innodb/t/innodb.opt

```diff
@@ -1 +1,2 @@
---innodb-defragment=0
\ No newline at end of file
+--innodb-defragment=0
+--loose-innodb_disable_resize_buffer_pool_debug=0
```

Comment:
I do not see `have_debug.inc` in `innodb.test`. Will the test still work for non-debug builds? Perhaps it would be better to write a separate test.

Reply:
The purpose of `loose` here is to work with debug/non-debug. Debug builds by default resist any changing of the buffer pool.

Reply:
Oh, I see it now. I somehow missed the `disable` in the name and thought that the purpose of the parameter was to disable the resizing, not to enable it.

I guess that this parameter was introduced in MySQL because buffer pool resizing in stress testing was revealing lots of trouble (genuine bugs that would affect buffer pool resizing while the server is under load). With some recent fixes, such as MDEV-26033, it should work more reliably.